System.IO.Abstractions.TestingHelpers/MockFile.cs

```diff
@@ -522,6 +584,18 @@ public override string ReadAllText(string path, Encoding encoding)
             return ReadAllTextInternal(path, encoding);
         }
 
+#if NETCOREAPP2_0
+        public async override Task<string> ReadAllTextAsync(string path, CancellationToken cancellationToken)
+        {
+            return ReadAllText(path);
```

Comment:
For these types of methods you can drop the `async` keyword and do a `return Task.FromResult(METHOD_CALL_HERE);` to save the overhead from the `async` keyword.

Reply:
Good suggestion! I'll make that happen - how do you suggest handling the *Async methods which return an empty `Task` (as they pass through to a `void`)?

Reply:
`Task.CompletedTask` is what you‘re for :)